pythonPackages.lektor: init at 2.3

ollieh opened the pull request:
###### Motivation for this change

Lektor was missing, first time packaging a python package, hope I did it properly.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [x] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Commit:
pythonPackages.lektor: init at 2.3

FRidh commented:
hi @ollieh 

Please try to follow the alphabetic 'order' that we try to maintain. It's a bit chaotic but you'll roughly find where the packages are supposed to be.

Furthermore, meta-data is missing.

Finally, have you tested whether the packages all build, and work for all interpreter versions?

ollieh commented:
Woops, my pypi mirror urls were 404ing, I guess because of the hash and because I'd already prefetched it, it must have just got the already downloaded one? Bit confusing that you have to put the first letter in a separate part of the URL. Fixed those, put it in alphabetical order, and added the metadata. Works on Python 2.7, 3.3, 3.5. Build seems to be failing on travis because of disk space.

FRidh commented:
Pushed 52bc8d203f846c0ac71af2080495a361ec84f1af.
In the future, please make sure that when tests are included, the tests actually run. If no tests are included disable them. Furthermore, each package should go in a commit of its own, which unfortunately I forgot to check before I pushed this.
